### PR TITLE
[develop] Reorganize verification tasks (rename tasks and files; split tasks into two)

### LINKED ISSUE #619: Rename vx tasks so they follow METplus-tool based convention
## Description

To avoid various sorts of file conflicts (e.g. multiple workflow tasks writing to the same output file), the eventual goal for the verification (vx) tasks in the ROCOTO workflow is for each task to only call one METplus tool (e.g. `GridStat`, `PointStat`, `EnsembleStat`, etc) and operate on one field or group of fields (see Issue #620 ).  For clarity, we would like the task names along with the corresponding j-job, ex-script, and log file names to reflect this.  

## Solution

In this issue, I propose that each item (task or file name) be renamed so that it contains the name of the METplus tool the task runs as well as the name of the field or group of fields on which it is run (e.g. 1-hour precip (APCP01h), reflectivity (REFC), etc).  

I propose to rename the j-job scripts as follows (note that NCO requires that all j-job names be in upper case):
```
JREGIONAL_RUN_VX_GRIDSTAT       --->  JREGIONAL_RUN_MET_GRIDSTAT_VX
JREGIONAL_RUN_VX_POINTSTAT      --->  JREGIONAL_RUN_MET_POINTSTAT_VX
JREGIONAL_RUN_VX_ENSGRID        --->  JREGIONAL_RUN_MET_ENSEMBLESTAT_VX_GRID
JREGIONAL_RUN_VX_ENSPOINT       --->  JREGIONAL_RUN_MET_ENSEMBLESTAT_VX_POINT
JREGIONAL_RUN_VX_ENSGRID_MEAN   --->  JREGIONAL_RUN_MET_GRIDSTAT_VX_ENSMEAN
JREGIONAL_RUN_VX_ENSPOINT_MEAN  --->  JREGIONAL_RUN_MET_POINTSTAT_VX_ENSMEAN
JREGIONAL_RUN_VX_ENSGRID_PROB   --->  JREGIONAL_RUN_MET_GRIDSTAT_VX_ENSPROB
JREGIONAL_RUN_VX_ENSPOINT_PROB  --->  JREGIONAL_RUN_MET_POINTSTAT_VX_ENSPROB
```
I propose to rename the ex-scripts as follows (note that NCO requires that all ex-script names be in lower case):
```
exregional_run_vx_ensgrid.sh        --->  exregional_run_met_ensemblestat_vx_grid.sh
exregional_run_vx_enspoint.sh       --->  exregional_run_met_ensemblestat_vx_point.sh
exregional_run_vx_gridstat.sh       --->  exregional_run_met_gridstat_vx.sh
exregional_run_vx_pointstat.sh      --->  exregional_run_met_pointstat_vx.sh
exregional_run_vx_ensgrid_mean.sh   --->  exregional_run_met_gridstat_vx_ensmean.sh
exregional_run_vx_enspoint_mean.sh  --->  exregional_run_met_pointstat_vx_ensmean.sh
exregional_run_vx_ensgrid_prob.sh   --->  exregional_run_met_gridstat_vx_ensprob.sh
exregional_run_vx_enspoint_prob.sh  --->  exregional_run_met_pointstat_vx_ensprob.sh
```
I propose to rename the METplus configuration files as follows (some aren't renamed):
```
EnsembleStat_APCP01h.conf      -->  EnsembleStat_APCP01h.conf      (no change)
EnsembleStat_APCP03h.conf      -->  EnsembleStat_APCP03h.conf      (no change)
EnsembleStat_APCP06h.conf      -->  EnsembleStat_APCP06h.conf      (no change)
EnsembleStat_APCP24h.conf      -->  EnsembleStat_APCP24h.conf      (no change)
EnsembleStat_REFC.conf         -->  EnsembleStat_REFC.conf         (no change)
EnsembleStat_RETOP.conf        -->  EnsembleStat_RETOP.conf        (no change)
EnsembleStat_conus_sfc.conf    -->  EnsembleStat_SFC.conf
EnsembleStat_upper_air.conf    -->  EnsembleStat_UPA.conf
GridStat_APCP01h.conf          -->  GridStat_APCP01h.conf          (no change)
GridStat_APCP03h.conf          -->  GridStat_APCP03h.conf          (no change)
GridStat_APCP06h.conf          -->  GridStat_APCP06h.conf          (no change)
GridStat_APCP24h.conf          -->  GridStat_APCP24h.conf          (no change)
GridStat_REFC.conf             -->  GridStat_REFC.conf             (no change)
GridStat_RETOP.conf            -->  GridStat_RETOP.conf            (no change)
PointStat_conus_sfc.conf       -->  PointStat_SFC.conf
PointStat_upper_air.conf       -->  PointStat_UPA.conf
GridStat_APCP01h_mean.conf     -->  GridStat_ensmean_APCP01h.conf
GridStat_APCP03h_mean.conf     -->  GridStat_ensmean_APCP03h.conf
GridStat_APCP06h_mean.conf     -->  GridStat_ensmean_APCP06h.conf
GridStat_APCP24h_mean.conf     -->  GridStat_ensmean_APCP24h.conf
GridStat_REFC_mean.conf        -->  GridStat_ensmean_REFC.conf
GridStat_RETOP_mean.conf       -->  GridStat_ensmean_RETOP.conf
PointStat_conus_sfc_mean.conf  -->  PointStat_ensmean_SFC.conf
PointStat_upper_air_mean.conf  -->  PointStat_ensmean_UPA.conf
GridStat_APCP01h_prob.conf     -->  GridStat_ensprob_APCP01h.conf
GridStat_APCP03h_prob.conf     -->  GridStat_ensprob_APCP03h.conf
GridStat_APCP06h_prob.conf     -->  GridStat_ensprob_APCP06h.conf
GridStat_APCP24h_prob.conf     -->  GridStat_ensprob_APCP24h.conf
GridStat_REFC_prob.conf        -->  GridStat_ensprob_REFC.conf
GridStat_RETOP_prob.conf       -->  GridStat_ensprob_RETOP.conf
PointStat_conus_sfc_prob.conf  -->  PointStat_ensprob_SFC.conf
PointStat_upper_air_prob.conf  -->  PointStat_ensprob_UPA.conf
```
Finally, I propose to rename the tasks and log file names for each of deterministic and ensemble verification as follows (e.g. for the 2019061500 cycle):

### Deterministic Verification
Rename the ROCOTO tasks as follows:
```
run_gridstatvx        --->  run_MET_GridStat_vx_APCP01h
run_gridstatvx_03h    --->  run_MET_GridStat_vx_APCP03h
run_gridstatvx_06h    --->  run_MET_GridStat_vx_APCP06h
run_gridstatvx_24h    --->  run_MET_GridStat_vx_APCP24h
run_gridstatvx_refc   --->  run_MET_GridStat_vx_REFC
run_gridstatvx_retop  --->  run_MET_GridStat_vx_RETOP
run_pointstatvx       --->  run_MET_PointStat_vx_SFC
                            + run_MET_PointStat_vx_UPA (split into two tasks)
```
Rename the corresponding log file for each task as follows:
```
run_gridstatvx_2019061500.log        --->  run_MET_GridStat_vx_APCP01h_2019061500.log
run_gridstatvx_03h_2019061500.log    --->  run_MET_GridStat_vx_APCP03h_2019061500.log
run_gridstatvx_06h_2019061500.log    --->  run_MET_GridStat_vx_APCP06h_2019061500.log
run_gridstatvx_24h_2019061500.log    --->  run_MET_GridStat_vx_APCP24h_2019061500.log
run_gridstatvx_refc_2019061500.log   --->  run_MET_GridStat_vx_REFC_2019061500.log
run_gridstatvx_retop_2019061500.log  --->  run_MET_GridStat_vx_RETOP_2019061500.log
run_pointstatvx_2019061500.log       --->  run_MET_PointStat_vx_SFC_2019061500.log
                                           + run_MET_PointStat_vx_UPA_2019061500.log (split into two files)
```
Rename the corresponding METplus log files as follows:
```
metplus.log.gridstat_2019061500_APCP_01h  --->  metplus.log.GridStat_APCP01h_2019061500
metplus.log.gridstat_2019061500_APCP_03h  --->  metplus.log.GridStat_APCP03h_2019061500
metplus.log.gridstat_2019061500_APCP_06h  --->  metplus.log.GridStat_APCP06h_2019061500
metplus.log.gridstat_2019061500_APCP_24h  --->  metplus.log.GridStat_APCP24h_2019061500
metplus.log.gridstat_2019061500_REFC      --->  metplus.log.GridStat_REFC_2019061500
metplus.log.gridstat_2019061500_RETOP     --->  metplus.log.GridStat_RETOP_2019061500
metplus.log.pointstat_2019061500_sfc      --->  metplus.log.PointStat_SFC_2019061500
metplus.log.pointstat_2019061500_upa      --->  metplus.log.PointStat_UPA_2019061500
```
### Ensemble Verification
Rename the ROCOTO tasks as follows:
```
run_gridstatvx_mem001        --->  run_MET_GridStat_vx_APCP01h_mem001
run_gridstatvx_03h_mem001    --->  run_MET_GridStat_vx_APCP03h_mem001
run_gridstatvx_06h_mem001    --->  run_MET_GridStat_vx_APCP06h_mem001
run_gridstatvx_24h_mem001    --->  run_MET_GridStat_vx_APCP24h_mem001
run_gridstatvx_refc_mem001   --->  run_MET_GridStat_vx_REFC_mem001
run_gridstatvx_retop_mem001  --->  run_MET_GridStat_vx_RETOP_mem001
run_pointstatvx_mem001       --->  run_MET_PointStat_vx_SFC_mem001
                                   + run_MET_PointStat_vx_UPA_mem001 (split into two tasks)

run_gridstatvx_mem002        --->  run_MET_GridStat_vx_APCP01h_mem002 
run_gridstatvx_03h_mem002    --->  run_MET_GridStat_vx_APCP03h_mem002
run_gridstatvx_06h_mem002    --->  run_MET_GridStat_vx_APCP06h_mem002
run_gridstatvx_24h_mem002    --->  run_MET_GridStat_vx_APCP24h_mem002
run_gridstatvx_refc_mem002   --->  run_MET_GridStat_vx_REFC_mem002
run_gridstatvx_retop_mem002  --->  run_MET_GridStat_vx_RETOP_mem002
run_pointstatvx_mem002       --->  run_MET_PointStat_vx_SFC_mem002
                                   + run_MET_PointStat_vx_UPA_mem002 (split into two tasks)

run_ensgridvx        --->  run_MET_EnsembleStat_vx_APCP01h
run_ensgridvx_03h    --->  run_MET_EnsembleStat_vx_APCP03h
run_ensgridvx_06h    --->  run_MET_EnsembleStat_vx_APCP06h
run_ensgridvx_24h    --->  run_MET_EnsembleStat_vx_APCP24h
run_ensgridvx_refc   --->  run_MET_EnsembleStat_vx_REFC
run_ensgridvx_retop  --->  run_MET_EnsembleStat_vx_RETOP
run_enspointvx       --->  run_MET_EnsembleStat_vx_SFC
                           + run_MET_EnsembleStat_vx_UPA (split into two tasks)

run_ensgridvx_mean      --->  run_MET_GridStat_vx_ensmean_APCP01h
run_ensgridvx_mean_03h  --->  run_MET_GridStat_vx_ensmean_APCP03h
run_ensgridvx_mean_06h  --->  run_MET_GridStat_vx_ensmean_APCP06h
run_ensgridvx_mean_24h  --->  run_MET_GridStat_vx_ensmean_APCP24h
run_enspointvx_mean     --->  run_MET_PointStat_vx_ensmean_SFC
                              + run_MET_PointStat_vx_ensmean_UPA (split into two tasks)

run_ensgridvx_prob        --->  run_MET_GridStat_vx_ensprob_APCP01h
run_ensgridvx_prob_03h    --->  run_MET_GridStat_vx_ensprob_APCP03h
run_ensgridvx_prob_06h    --->  run_MET_GridStat_vx_ensprob_APCP06h
run_ensgridvx_prob_24h    --->  run_MET_GridStat_vx_ensprob_APCP24h
run_ensgridvx_prob_refc   --->  run_MET_GridStat_vx_ensprob_REFC
run_ensgridvx_prob_retop  --->  run_MET_GridStat_vx_ensprob_RETOP
run_enspointvx_prob       --->  run_MET_PointStat_vx_ensprob_SFC
                                + run_MET_PointStat_vx_ensprob_UPA (split into two tasks)
```
Rename the corresponding log file for each task as follows:
```
run_gridstatvx_mem001_2019061500.log        --->  run_MET_GridStat_vx_APCP01h_mem001_2019061500.log
run_gridstatvx_03h_mem001_2019061500.log    --->  run_MET_GridStat_vx_APCP03h_mem001_2019061500.log
run_gridstatvx_06h_mem001_2019061500.log    --->  run_MET_GridStat_vx_APCP06h_mem001_2019061500.log
run_gridstatvx_24h_mem001_2019061500.log    --->  run_MET_GridStat_vx_APCP24h_mem001_2019061500.log
run_gridstatvx_refc_mem001_2019061500.log   --->  run_MET_GridStat_vx_REFC_mem001_2019061500.log
run_gridstatvx_retop_mem001_2019061500.log  --->  run_MET_GridStat_vx_RETOP_mem001_2019061500.log
run_pointstatvx_mem001_2019061500.log       --->  run_MET_PointStat_vx_SFC_mem001_2019061500.log
                                                  + run_MET_PointStat_vx_UPA_mem001_2019061500.log (split into two files)

run_gridstatvx_mem002_2019061500.log        --->  run_MET_GridStat_vx_APCP01h_mem002_2019061500.log
run_gridstatvx_03h_mem002_2019061500.log    --->  run_MET_GridStat_vx_APCP03h_mem002_2019061500.log
run_gridstatvx_06h_mem002_2019061500.log    --->  run_MET_GridStat_vx_APCP06h_mem002_2019061500.log
run_gridstatvx_24h_mem002_2019061500.log    --->  run_MET_GridStat_vx_APCP24h_mem002_2019061500.log
run_gridstatvx_refc_mem002_2019061500.log   --->  run_MET_GridStat_vx_REFC_mem002_2019061500.log
run_gridstatvx_retop_mem002_2019061500.log  --->  run_MET_GridStat_vx_RETOP_mem002_2019061500.log
run_pointstatvx_mem002_2019061500.log       --->  run_MET_PointStat_vx_SFC_mem002_2019061500.log
                                                  + run_MET_PointStat_vx_UPA_mem002_2019061500.log (split into two files)

run_ensgridvx_2019061500.log        --->  run_MET_EnsembleStat_vx_APCP01h_2019061500.log
run_ensgridvx_03h_2019061500.log    --->  run_MET_EnsembleStat_vx_APCP03h_2019061500.log
run_ensgridvx_06h_2019061500.log    --->  run_MET_EnsembleStat_vx_APCP06h_2019061500.log
run_ensgridvx_24h_2019061500.log    --->  run_MET_EnsembleStat_vx_APCP24h_2019061500.log
run_ensgridvx_refc_2019061500.log   --->  run_MET_EnsembleStat_vx_REFC_2019061500.log
run_ensgridvx_retop_2019061500.log  --->  run_MET_EnsembleStat_vx_RETOP_2019061500.log
run_enspointvx_2019061500.log       --->  run_MET_EnsembleStat_vx_SFC_2019061500.log
                                          + run_MET_EnsembleStat_vx_UPA_2019061500.log (split into two files)

run_ensgridvx_mean_2019061500.log      --->  run_MET_GridStat_vx_ensmean_APCP01h_2019061500.log
run_ensgridvx_mean_03h_2019061500.log  --->  run_MET_GridStat_vx_ensmean_APCP03h_2019061500.log
run_ensgridvx_mean_06h_2019061500.log  --->  run_MET_GridStat_vx_ensmean_APCP06h_2019061500.log
run_ensgridvx_mean_24h_2019061500.log  --->  run_MET_GridStat_vx_ensmean_APCP24h_2019061500.log
run_enspointvx_mean_2019061500.log     --->  run_MET_PointStat_vx_ensmean_SFC_2019061500.log
                                             + run_MET_PointStat_vx_ensmean_UPA_2019061500.log (split into two files)

run_ensgridvx_prob_2019061500.log        --->  run_MET_GridStat_vx_ensprob_APCP01h_2019061500.log
run_ensgridvx_prob_03h_2019061500.log    --->  run_MET_GridStat_vx_ensprob_APCP03h_2019061500.log
run_ensgridvx_prob_06h_2019061500.log    --->  run_MET_GridStat_vx_ensprob_APCP06h_2019061500.log
run_ensgridvx_prob_24h_2019061500.log    --->  run_MET_GridStat_vx_ensprob_APCP24h_2019061500.log
run_ensgridvx_prob_refc_2019061500.log   --->  run_MET_GridStat_vx_ensprob_REFC_2019061500.log
run_ensgridvx_prob_retop_2019061500.log  --->  run_MET_GridStat_vx_ensprob_RETOP_2019061500.log
run_enspointvx_prob_2019061500.log       --->  run_MET_PointStat_vx_ensprob_SFC_2019061500.log
                                               + run_MET_PointStat_vx_ensprob_UPA_2019061500.log (split into two files)
```
Rename the corresponding METplus log files as follows:
```
metplus.log.gridstat_2019061500_mem001_APCP_01h  --->  metplus.log.GridStat_APCP01h_mem001_2019061500
metplus.log.gridstat_2019061500_mem001_APCP_03h  --->  metplus.log.GridStat_APCP03h_mem001_2019061500
metplus.log.gridstat_2019061500_mem001_APCP_06h  --->  metplus.log.GridStat_APCP06h_mem001_2019061500
metplus.log.gridstat_2019061500_mem001_APCP_24h  --->  metplus.log.GridStat_APCP24h_mem001_2019061500
metplus.log.gridstat_2019061500_mem001_REFC      --->  metplus.log.GridStat_REFC_mem001_2019061500
metplus.log.gridstat_2019061500_mem001_RETOP     --->  metplus.log.GridStat_RETOP_mem001_2019061500
metplus.log.pointstat_2019061500_mem001_sfc      --->  metplus.log.PointStat_SFC_mem001_2019061500
metplus.log.pointstat_2019061500_mem001_upa      --->  metplus.log.PointStat_UPA_mem001_2019061500

metplus.log.gridstat_2019061500_mem002_APCP_01h  --->  metplus.log.GridStat_APCP01h_mem002_2019061500
metplus.log.gridstat_2019061500_mem002_APCP_03h  --->  metplus.log.GridStat_APCP03h_mem002_2019061500
metplus.log.gridstat_2019061500_mem002_APCP_06h  --->  metplus.log.GridStat_APCP06h_mem002_2019061500
metplus.log.gridstat_2019061500_mem002_APCP_24h  --->  metplus.log.GridStat_APCP24h_mem002_2019061500
metplus.log.gridstat_2019061500_mem002_REFC      --->  metplus.log.GridStat_REFC_mem002_2019061500
metplus.log.gridstat_2019061500_mem002_RETOP     --->  metplus.log.GridStat_RETOP_mem002_2019061500
metplus.log.pointstat_2019061500_mem002_sfc      --->  metplus.log.PointStat_SFC_mem002_2019061500
metplus.log.pointstat_2019061500_mem002_upa      --->  metplus.log.PointStat_UPA_mem002_2019061500

metplus.log.ensgrid_2019061500_APCP_01h  --->  metplus.log.EnsembleStat_APCP01h_2019061500
metplus.log.ensgrid_2019061500_APCP_03h  --->  metplus.log.EnsembleStat_APCP03h_2019061500
metplus.log.ensgrid_2019061500_APCP_06h  --->  metplus.log.EnsembleStat_APCP06h_2019061500
metplus.log.ensgrid_2019061500_APCP_24h  --->  metplus.log.EnsembleStat_APCP24h_2019061500
metplus.log.ensgrid_2019061500_REFC      --->  metplus.log.EnsembleStat_REFC_2019061500
metplus.log.ensgrid_2019061500_RETOP     --->  metplus.log.EnsembleStat_RETOP_2019061500
metplus.log.enspoint_2019061500_sfc      --->  metplus.log.EnsembleStat_SFC_2019061500
metplus.log.enspoint_2019061500_upa      --->  metplus.log.EnsembleStat_UPA_2019061500

metplus.log.ensgrid_mean_2019061500_APCP_01h  --->  metplus.log.GridStat_ensmean_APCP01h_2019061500
metplus.log.ensgrid_mean_2019061500_APCP_03h  --->  metplus.log.GridStat_ensmean_APCP03h_2019061500
metplus.log.ensgrid_mean_2019061500_APCP_06h  --->  metplus.log.GridStat_ensmean_APCP06h_2019061500
metplus.log.enspoint_mean_2019061500_sfc      --->  metplus.log.PointStat_ensmean_SFC_2019061500
metplus.log.enspoint_mean_2019061500_upa      --->  metplus.log.PointStat_ensmean_UPA_2019061500

metplus.log.ensgrid_prob_2019061500_APCP_01h  --->  metplus.log.GridStat_ensprob_APCP01h_2019061500
metplus.log.ensgrid_prob_2019061500_APCP_03h  --->  metplus.log.GridStat_ensprob_APCP03h_2019061500
metplus.log.ensgrid_prob_2019061500_APCP_06h  --->  metplus.log.GridStat_ensprob_APCP06h_2019061500
metplus.log.ensgrid_prob_2019061500_REFC      --->  metplus.log.GridStat_ensprob_REFC_2019061500
metplus.log.ensgrid_prob_2019061500_RETOP     --->  metplus.log.GridStat_ensprob_RETOP_2019061500
metplus.log.enspoint_prob_2019061500_sfc      --->  metplus.log.PointStat_ensprob_SFC_2019061500
metplus.log.enspoint_prob_2019061500_upa      --->  metplus.log.PointStat_ensprob_UPA_2019061500
```

## Acceptance Criteria (Definition of Done)

WE2E verification tests run without changing results.

## Alternative Solutions (optional)

If you have other ideas for a naming convention, please suggest in the discussion below.

### PR DESCRIPTION
**Pull request by gsketefian**
## DESCRIPTION OF CHANGES: 

1) Rename verification tasks and corresponding j-jobs, ex-scripts, and log files as described in Issue #619.
2) Separate combined METplus vx tasks for surface and upper air into separate tasks.  This is so that there is only one call to a METplus tool per ROCOTO task and helps simplify the workflow and debugging.
3) Ensure that each vx task has a separate section in `config_defaults.yaml` with its own resource variables (`WTIME_...`, `PPN_...`, etc).

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## TESTS CONDUCTED: 

- [x] hera.intel
- [ ] orion.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [ ] gaea.intel
- [ ] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [x] fundamental test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

I have run the following fundamental WE2E tests on Hera with the Intel compiler.  All were successful.  Also, all task names and log file names are as described in Issue #619.
```
MET_verification_only_preproc_vx
MET_verification_only_preproc_makeics_vx
MET_verification
MET_verification_only_vx
community_ensemble_2mems
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp_regional_plot
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR
grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v15p2
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp_regional_plot
MET_ensemble_verification
community_ensemble_2mems_stoch
pregen_grid_orog_sfc_climo
```

## DOCUMENTATION:

This PR requires changes to the documentation.  However, **I have not yet made changes to the documentation.  There will likely be further changes required to the documentation due to several upcoming vx PRs.  I prefer to make all the documentation changes in one go after all the vx PRs have been merged.**

## ISSUE: 

This resolves Issue #619.

## CHECKLIST

- [x] My code follows the style guidelines in the Contributor's Guide
- [x] I have performed a self-review of my own code using the Code Reviewer's Guide
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain).
- [ ] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## LABELS (optional): 

A Code Manager needs to add the following labels to this PR: 
- [ ] Work In Progress
- [ ] bug
- [ ] enhancement
- [ ] documentation
- [ ] release
- [ ] high priority
- [ ] run_ci
- [ ] run_we2e_fundamental_tests
- [ ] run_we2e_comprehensive_tests
- [ ] Needs Cheyenne test 
- [ ] Needs Jet test 
- [x] Needs Hera test 
- [ ] Needs Orion test 
- [ ] help wanted

## CONTRIBUTORS (optional): 

@michelleharrold @JeffBeck-NOAA @willmayfield @mkavulich

### CODE REVIEW

**Comment by gsketefian** [comment]
@MichaelLueken @gspetro Just wanted to draw your attention to the fact that I haven't yet made the documentation changes that this PR requires.  That's mostly because those changes will need to be modified yet again due to additonal vx PRs coming in the very near future.  Is it ok with you to put off the doc changes until all the vx PRs are in (hopefully all this month)?  Otherwise I'm happy to make them.

**Comment by MichaelLueken** [comment]
@gsketefian If the documentation will change as more verification PRs are introduced, I think it would be fine to hold off on updating the documentation until the PRs that would require additional modifications have been introduced.  You should create an issue to let everyone know that you are planning on updating the documentation, but are waiting until the rest of the verification PRs are in before moving forward with it.

File: jobs/JREGIONAL_RUN_MET_ENSEMBLESTAT_VX_GRID
Comment (danielabdi-noaa) [inline jobs/JREGIONAL_RUN_MET_ENSEMBLESTAT_VX_GRID]: @gsketefian NCO requires that ex-scripts be named all lowercase so this and others need to change.

File: ush/config_defaults.yaml
Comment (danielabdi-noaa) [inline ush/config_defaults.yaml]: To be consistent with the modified all lowercase ex-script name of the task, this also needs to be all in lowercase.

File: ush/config_defaults.yaml
Comment (danielabdi-noaa) [inline ush/config_defaults.yaml:2183]: The name of the task should probably change too just for consistency.

File: jobs/JREGIONAL_RUN_MET_ENSEMBLESTAT_VX_GRID
Comment (gsketefian) [reply jobs/JREGIONAL_RUN_MET_ENSEMBLESTAT_VX_GRID]: @danielabdi-noaa Ok, ex-script names have been changed, will push commit soon.

File: ush/config_defaults.yaml
Comment (gsketefian) [reply ush/config_defaults.yaml]: @danielabdi-noaa Changed these as well.

File: ush/config_defaults.yaml
Comment (gsketefian) [reply ush/config_defaults.yaml:2183]: @danielabdi-noaa Since there is no NCO requirement on the ROCOTO task names, I decided to keep them as a mix of upper and lower case in order to match the names of the log files as described in Issue #618 (and since to me this naming convention is easier to read, which is my reason to use it in the first place).  I will ask in that issue whether people have a preference on this.

**Comment by gsketefian** [comment]
@MichaelLueken @gspetro I just created Issue #630 for updating the vx documentation.

**Comment by MichaelLueken** [comment]
Thank you very much, @gsketefian, for creating issue #630!

**Review by danielabdi-noaa — APPROVED** [review]
@gsketefian Thanks for making the changes, approving now.

**Review by MichaelLueken — APPROVED** [review]
@gsketefian Thank you very much for addressing @danielabdi-noaa's concerns!  Following the renaming of the ex- scripts to be lower case, per NCO guidelines, these changes look good to me.  I will go ahead and approve these changes now.

**Comment by gsketefian** [comment]
@danielabdi-noaa @MichaelLueken Thanks for the approvals!  

I am retesting now on Hera to make sure all the fundamental tests pass.  Also, I'd like to wait until tomorrow (morning, let's say) to get some feedback on Issue #619 (so far I haven't gotten any) before proceeding further.  Assuming my tests pass and there is no or minor feedback on #619 by tomorrow, could @MichaelLueken launch the automated tests then?  Thanks.

**Comment by MichaelLueken** [comment]
> @danielabdi-noaa @MichaelLueken Thanks for the approvals!
> 
> I am retesting now on Hera to make sure all the fundamental tests pass. Also, I'd like to wait until tomorrow (morning, let's say) to get some feedback on Issue #619 (so far I haven't gotten any) before proceeding further. Assuming my tests pass and there is no or minor feedback on #619 by tomorrow, could @MichaelLueken launch the automated tests then? Thanks.

@gsketefian After I gave my approval, I went ahead and submitted the Jenkins tests.  If any additional changes are pushed to your `feature/rename_vx_tasks` branch, I'll try and rerun the automated tests when I get back from jury duty tomorrow.

**Comment by gsketefian** [comment]
@MichaelLueken Ok, thanks for letting me know.  One question -- does the automated testing system first merge `develop` into the PR and then run the tests?  I just realized I haven't yet merged in `develop`.

**Comment by MichaelLueken** [comment]
@gsketefian Yes, the first thing that the automated Jenkins tests do is merge the latest develop branch into the branch associated with the PR.  So, no worries with that.  If discussion from issue #619 leads to additional modifications, then the tests will need to be rerun, but you don't need to worry about the PR being up-to-date with develop until just before merging.

**Comment by venitahagerty** [comment]
Machine: hera
Compiler: intel
Job: WE
Repo location: /scratch1/BMC/zrtrr/rrfs_ci/autoci/pr/1245532184/20230227205016/ufs-srweather-app
Build was Successful
Rocoto jobs started
Long term tracking will be done on 10 experiments
If test failed, please make changes and add the following label back:
ci-hera-intel-WE
Experiment Succeeded on hera: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp_regional_plot
Experiment Failed on hera: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
2023-02-27 22:16:06 +0000 :: hfe10 :: Task make_ics, jobid=42409022, in state DEAD (FAILED), ran for 40.0 seconds, exit status=256, try=2 (of 2)
Experiment Succeeded on hera: community_ensemble_2mems_stoch
Experiment Failed on hera: grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR
2023-02-27 22:12:09 +0000 :: hfe07 :: Task make_ics, jobid=42408698, in state DEAD (FAILED), ran for 39.0 seconds, exit status=256, try=2 (of 2)
Experiment Succeeded on hera: pregen_grid_orog_sfc_climo
Experiment Succeeded on hera: grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v15p2
Experiment Succeeded on hera: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2
Experiment Succeeded on hera: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR
Experiment Succeeded on hera: grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
Experiment Succeeded on hera: MET_ensemble_verification
All experiments completed

**Comment by gsketefian** [comment]
@MichaelLueken Ok, thanks!

**Comment by gsketefian** [comment]
@MichaelLueken All the fundamental tests I ran locally on Hera passed.

**Comment by gsketefian** [comment]
@MichaelLueken FYI unfortunately I found bugs in a couple of the vx ex-scripts.  They only affect verification.  I fixed them and reran the three vx tests (`MET_verification`, `MET_verification_only_vx`, and `MET_ensemble_verification`).  They were successful, so I pushed the changes to the PR.

**Comment by gsketefian** [comment]
@MichaelLueken A couple of people are going to take a look at Issue #619 by COB today, so could you hold off on rerunning the automated tests until then?  I can let you know when the PR is ready for the re-testing.  Thanks!

**Comment by MichaelLueken** [comment]
@gsketefian I'm on PTO today, so if it is okay with you, I will resubmit the automated tests first thing tomorrow morning.  They should run fast enough so that this work can be merged early in the day.  Thanks!

**Comment by gsketefian** [comment]
> @gsketefian I'm on PTO today, so if it is okay with you, I will resubmit the automated tests first thing tomorrow morning. They should run fast enough so that this work can be merged early in the day. Thanks!

@MichaelLueken Yes, that sounds good.

**Comment by MichaelLueken** [comment]
@gsketefian I have just resubmitted the automated Jenkins tests.  Are there any other changes that you will be making, or will these changes be ready to be merged once the tests successfully pass?

**Comment by gsketefian** [comment]
@MichaelLueken No more changes, so please go ahead and merge it if the tests pass.  Thanks!

**Comment by MichaelLueken** [comment]
@gsketefian Great!  I will move forward with these changes once they pass the automated tests.

**Pull request merged**
